storybook/pages/EnsViewPage.qml

```diff
@@ -0,0 +1,313 @@
+                    }
+                    return ensUsernameModule.usernameAvailabilityChecked("available")
+                }
+
+                function getChainIdForEns() {
+                    logs.logEvent("ensUsernamesStore::getChainIdForEns")
+                    return "chainId"
+                }
+
+                function setPubKeyGasEstimate(address) {
+                    logs.logEvent("ensUsernamesStore::setPubKeyGasEstimate", ["address"], arguments)
+                    return 100 // gas estimate
+                }
+
+                function setPubKey(ensUsername, address, gas, gasPrice, maxPriorityFeePerGas, maxFeePerGas, password, eip1559Enabled) {
+                    logs.logEvent("ensUsernamesStore::setPubKey", ["ensUsername", "address", "gas", "gasPrice", "maxPriorityFeePerGas", "maxFeePerGas", "password", "eip1559Enabled"], arguments)
+                    return "0x123"
+                }
+
+                function ensConnectOwnedUsername(name, isStatus) {
+                    logs.logEvent("ensUsernamesStore::ensConnectOwnedUsername", ["name", "isStatus"], arguments)
+                }
+
+                function getEnsRegisteredAddress() {
+                    logs.logEvent("ensUsernamesStore::getEnsRegisteredAddress")
+                    return "0x234"
+                }
+
+                function registerEnsGasEstimate(username, address) {
+                    logs.logEvent("ensUsernamesStore::registerEnsGasEstimate", ["username", "address"], arguments)
+                    return 100
+                }
+
+                function registerEns(ensUsername, address, gasLimit, gasPrice, tipLimit, overallLimit, password, eip1559Enabled) {
+                    logs.logEvent("ensUsernamesStore::registerEns", ["ensUsername", "address", "gasLimit", "gasPrice", "tipLimit", "overallLimit", "password", "eip1559Enabled"], arguments)
+                }
+
+                function getSntBalance() {
+                    logs.logEvent("ensUsernamesStore::getSntBalance")
+                    return 1000
+                }
+
+                function copyToClipboard(pubKey) {
+                    logs.logEvent("ensUsernamesStore::copyToClipboard", ["pubKey"], arguments)
+                }
+
+                function getWalletDefaultAddress() {
+                    logs.logEvent("ensUsernamesStore::getWalletDefaultAddress")
+                    return "0x345"
+                }
+
+                function getEtherscanLink() {
+                    logs.logEvent("ensUsernamesStore::getEtherscanLink")
+                    return "http://etherscan.com"
+                }
+
+                function getEnsRegistry() {
+                    logs.logEvent("ensUsernamesStore::getEnsRegistry")
+                    return "0x456"
+                }
+            }
+
+            contactsStore: QtObject {
+            }
+
+            stickersStore: QtObject {
+                property var stickersModule: QtObject {
```

Comment:
if it's just a "placeholder" you can add `property QtObject stickersModule`

Reply:
a plain `property var stickersModule` would do as well :)